client.js

```diff
@@ -103,21 +102,44 @@ function createReporter() {
     overlay = require('./client-overlay');
   }
 
-
+  var styles = {
```

Comment:
OK! Actually, I was intended to do that, but I found a little trouble to merge `master` and `multi-compiler`.

Anyway, I will try it again.